### ISSUE TITLE
Add support Kubernetes Provider

### ISSUE THREAD

**Issue by sergeylanzman** [GoogleCloudPlatform/terraformer#3]
Kubernetes provider can be supported by terraformer. 
Use kubernetes go client for get IDs.
Use kubectl config default file.

docs: https://www.terraform.io/docs/providers/kubernetes/index.html

**Comment by pdecat** [comment]
On the kubernetes YAML to terraform configuration generation front, there's https://github.com/sl1pm4t/k2tf which is quite effective.

**Comment by hamed1soleimani** [comment]
I'm going for implementing it. to make `GetSupportedService` more flexible, we can use client-go discovery methods such as `ServerPreferredResources`. Then It's quite straightforward to extract resources of each type via clientset apis.

**Comment by sergeylanzman** [comment]
Ok GetSupportedService it's part of interface, you can change implementation in kubernetes provider.
GetSupportedService - return list of terraformer support services, not provider.

**Comment by hamed1soleimani** [comment]
Yeah, I was saying about the supported services as well. The `ServerPreferredResources` will return list of resources that are supported by k8s server. I was saying instead of hardcoding the list of supported k8s resources, I can use client-go to return a list of available k8s resources from k8s api-server.
